packages/perseus/src/widgets/interactive-graphs/graphs/polygon.tsx

```diff
@@ -113,7 +113,6 @@ export const PolygonGraph = (props: Props) => {
                     onMove={(destination: vec.Vector2) =>
                         dispatch(movePoint(i, destination))
                     }
-                    data-testid={type + i}
```

Comment:
Same question here

Reply:
This wasn't used in any tests so I thought it was better to remove them for now. If future tests need them we can bring back the test ids.